src/Microsoft.AspNet.Mvc/MvcServiceCollectionExtensions.cs

```diff
@@ -44,7 +44,6 @@ public static IMvcBuilder AddMvc(this IServiceCollection services, Action<MvcOpt
 
             builder.AddApiExplorer();
             builder.AddAuthorization();
-
             // Order added affects options setup order
```

Comment:
Might be due to the rebase? I don't recall changing that, still the change looks right and is harmless